frontend/src/components/TaskView/FutureView/FutureViewDayTaskContainer.module.css

```diff
@@ -1,3 +1,4 @@
 .Container {
   height: calc(100% - 66px);
+  min-height: 216px;
```

Comment:
This is problematic. Things might only work for certain screen size.

Reply:
we can try to calculate the container height instead, or an easier way to deal with this is to just disable drag and drop for 2W and monthly views